website/src/pages/showcase/index.tsx

```diff
@@ -31,6 +32,12 @@ const DESCRIPTION = 'List of websites people are building with Docusaurus';
 const EDIT_URL =
   'https://github.com/facebook/docusaurus/edit/main/website/src/data/users.tsx';
 
+function setScrollTopPosition(position: number = 0) {
+  if (ExecutionEnvironment.canUseDOM) {
```

Comment:
Why not `useIsBrowser()` though, because that's more React-y? And even re-use the logic from `theme-common`? I'm actually confused by all these APIs we have😅

Reply:
As far as I understand, `useIsBrowser` should be used directly in React context, but `ExecutionEnvironment` is meant for util functions like `setScrollTopPosition`.

Reply:
useIsBrowser is very important to avoid hydration issues if the result leads to a different JSX rendered the first time.

After hydration (ie in callbacks) it does not matter anymore